docs/create_robots_whoami.md

```diff
@@ -8,20 +8,25 @@ Your robot's `whoami` package serves as a configuration package for the `rai_who
 
 ## Example (Franka Emika Panda arm)
 
-1. Create a whoami package for Panda
+1. Setup the repository using 1st and 2nd step from [Setup](../README.md#setup)
 
-```
-poetry run create_rai_ws --name panda --destination-directory src/examples
-```
+2. Create a whoami package for Panda
 
-2. Fill in the `src/examples/panda_whoami/description` folder with data:\
+   ```shell
+   poetry run create_rai_ws --name panda --destination-directory src/examples
+   ```
+
+3. Fill in the `src/examples/panda_whoami/description` folder with data:\
    2.1 Save [this image](https://robodk.com/robot/img/Franka-Emika-Panda-robot.png) into `src/examples/panda_whoami/description/images`\
    2.2 Save [this document](https://github.com/user-attachments/files/16417196/Franka.Emika.Panda.robot.-.RoboDK.pdf) in `src/examples/panda_whoami/description/documentation`
 
-3. Run the `parse_whoami_package`. This will process the documentation, building it into a vector database, which is used by RAI agent to reason about its identity.
+4. Run the `parse_whoami_package`. This will process the documentation, building it into a vector database, which is used by RAI agent to reason about its identity.
 
-```
-poetry run parse_whoami_package src/examples/panda_whoami/description src/examples/panda_whoami/description
+> **NOTE**: Vector database is created using the OpenAI API. Parsing bigger documents
```

Comment:
Not true, the vendor is used based on config.toml

Reply:
https://github.com/RobotecAI/rai/pull/239/commits/44392f0fe3ac1ca494f87beb0f5523b1e3c7b805

Reply:
Didn't know! I tested `ollama` and it worked! Docs adjusted